src/Application.php

```diff
@@ -100,8 +103,13 @@ public function migrateTransformationConfig(array $transformationConfig): array
 
             $result[] = [
                 'componentId' => $resultTransformationV2->getComponentId(),
-                'id' => $newConfig['id'],
+                'id' => $newConfig->getConfigurationId(),
             ];
+
+            try {
+                $this->writeConfigMetadata($config, $newConfig, $transformationConfig['name']);
+            } catch (Throwable $e) {
```

Comment:
Nechceme zalogovat aspon nejaku debug hlasku? 
To sa bude velmi blbo debugovat, ak to bude padat prave tu.

Reply:
jsou to všechno sync akce, takže něco logovat je zbytečný => nikam se to stejně nepropíše

Reply:
ok, to mi nedoslo, to by sa musel pouzit gelf logger.
A nechces radsej tu chybu vratit uzivatelovi, alebo spravit `Application error`.
Malo by to vzdy prejst, nie?

Reply:
(pripadne retry)

Reply:
možná asi to vrátíme uživateli... původně jsem přemýšlel tak že pokud tam nastane chyba, tak se jen nezápíše transformace do folderu... ale asi to bude pro klienta matoucí

Reply:
@ondrajodas budes to este upravovat? som tu na review, ale myslel som, ze toto chces este doriesit ...

Reply:
joo určitě... na review jsem tě znovu dával v pátek před druhým komentářem... zatím tě dám přyč a až to bude tak tě tam dám znovu

Reply:
ok

Reply:
musím kvůli tomu trochu předělat testy, abych ten try/catch mohl otestovat

Reply:
https://github.com/keboola/app-transformation-migrate/pull/41/commits/045c97fc7db62faef95beb07b3773e98cb74eb5b